daily_milestone.go

```diff
@@ -132,6 +184,24 @@ func main() {
 	// Initializing logger
 	LoggerSetup(os.Stdout)
 	// Calling getProjectID
-	getProjectID(APIBase, Token, Project, Namespace)
-	createMilestoneData(Advance)
+	if Token == "" || APIBase == "" || Namespace == "" || Project == "" {
+		message := "Please provide all necessary Gitlab settings. \n See --help for more information."
+		logger.Println(message)
+	} else {
+		projectID := getProjectID(APIBase, Token, Project, Namespace)
+		newMilestone := getMilestones(APIBase, Token, projectID)
+		oldMilestone := createMilestoneData(Advance)
+
+		for index, y := range newMilestone {
+			for _, z := range oldMilestone {
+				if z == y {
+					newMilestone = append(newMilestone[:index], newMilestone[(index+1):]...)
```

Comment:
This method could potentially lead to memory leaks, see https://github.com/golang/go/wiki/SliceTricks 
(However this is more about warning you about the possibility, for the scope of this tool this is probably fine)